### ISSUE TITLE
mavenCentral not required

### ISSUE THREAD

**Issue by felixwiemuth** [cketti/ckChangeLog#42]
It should not be necessary anymore to add `mavenCentral()` to the repositories in the build scripts as indicated in the Readme. AndroidStudio project by default include `jCenter()` which replaces `mavenCentral()`.

**Comment by cketti** [comment]
jcenter mirrors most of the packages from Maven Central. So right now things work without adding `mavenCentral()`. But I consider that to be an implementation detail of jcenter, not something that will always be true. I only upload my libraries to Maven Central.

**Comment by cketti** [comment]
However, I'm open to changing the language in the README to make adding `mavenCentral()` optional. Suggestions are welcome.

**Comment by felixwiemuth** [comment]
Yeah, I guess simply changing the text to sth. like "add `jCenter()` or alternatively `mavenCentral()` to ... in case the library is not found" saves developers a bit time.
